web/src/unread_ui.ts

```diff
@@ -54,7 +54,8 @@ export function update_unread_banner(): void {
 export function hide_unread_banner(): void {
     // Use visibility instead of hide() to prevent messages on the screen from
```

Comment:
I think you missed this comment. We would want to remove this if we have shifted from "invisible" to "hide".

Reply:
I'm not sure I quite understand the shift you're talking about, but we don't want weird unexpected.

Reply:
I think @roanster007  is referring to the layout shift caused by the 'Mark as read' banner appearing between the 'Someone is typing...' text and the last message. I feel that is fine since 'Mark as unread' is a user-enabled action, a conscious choice and not something someone would do often.

Reply:
ya that's what I was referring to.